### PR TITLE
Fix/replace strings

### LINKED ISSUE #106: Erase strings in logging messages as variables
If you replace that like this, the actual parameter in the CSV can differ from the one in the log message. You should change to 'f"Setting {SENSITIVITY_ALL_COMBINATIONS}..."'

https://github.com/rl-institut/offgridders/pull/89/files#diff-7d6f671499387ff9c25d94f63387847dL66

### LINKED ISSUE #107: Change strings to variables in paths
Join using os.join module

### LINKED ISSUE #115: Create variables for filenames
For example in 

src/C_sensitivity_experiments.py

Change "/si..csv" by a variable 

settings[OUTPUT_FOLDER] + "/simulation_experiments.csv"

### LINKED ISSUE #116: Remove variables for graph description in src/G4_output_functions.py
WIND: "#4ca7c3"
  PV: "#ffde32"

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
This should be a more complete version of the replace_strings branch. Only in H1 are strings left, for they are widely used. The string "oemof" as it appears in different variations was ignored in this PR. 
This should fix #116 , fix #115, fix #107, fix #106

### CODE REVIEW

File: src/H1_multicriteria_functions.py [hunk 1]
```diff
@@ -864,9 +885,9 @@ def plot_evaluations(
 
     cases_exp = []
     for i in range(combinations):
-        cases_exp.append("case_" + str(i + 1))
+        cases_exp.append(PREFIX_CASE + str(i + 1))
 
-    if os.path.isdir(settings[OUTPUT_FOLDER] + MCA_PLOTS):
+    if os.path.isdir(os.path.join(settings[OUTPUT_FOLDER] + MCA_PLOTS)):
```
Comment (santiagoinfantinom) [inline src/H1_multicriteria_functions.py:890]: Correct this! os.path.join(settings,MCA)

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ Here is a template for new release sections
 
 ### Added
 -Constant variables for strings in constants.py (#89)
```
Comment (smartie2076) [inline CHANGELOG.md:24]: Where is the changelog entry for this PR?

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ Here is a template for new release sections
...
+-CSV,PNG files and paths added (#117)
```
Comment (smartie2076) [inline CHANGELOG.md:25]:
```suggestion
-CSV,PNG files and paths now defined by variables in `constants.py` (#117)
```

File: CHANGELOG.md (hunk 2 above)
Comment (smartie2076) [reply CHANGELOG.md:24]: If multiple PR work on the same issue mentioned in the changelog, you would list the PRs in the brackets (eg. (89, 117, 118).

File: src/B_read_from_files.py [hunk 4]
```diff
@@ -443,7 +443,7 @@ def check_output_directory(settings, input_excel_file):
     os.mkdir(output_folder + INPUTS_FOLDER)
 
     path_from = os.path.abspath(input_excel_file)
-    path_to = os.path.abspath(output_folder + INPUT_TEMPLATE_EXCEL_XLSX)
+    path_to = os.path.abspath(os.path.join(output_folder + INPUT_TEMPLATE_EXCEL_XLSX))
```
Comment (smartie2076) [inline src/B_read_from_files.py:467]:
```suggestion
    path_to = os.path.abspath(os.path.join(output_folder, INPUT_TEMPLATE_EXCEL_XLSX))
```
os.path.join replaces you adding strings and slashes. So, you should have:
`os.path.join(folder_name, file_name). You need to replace the `+` with `,` and remove the slash from `INPUT_TEMPLATE_EXCEL_XLSX` and the like.

**Comment by santiagoinfantinom** [comment]
#117 was a PR from **fix/strings_left** to **fix/replace_strings** to make the review easier. The merging of this PR should close the open Issues mentioned above, for they wouldn't close after merging **fix/strings_left**  with **fix/replace_strings**

File: CHANGELOG.md (hunk 3 above)
Comment (santiagoinfantinom) [reply CHANGELOG.md]: I put all PRs related to this in the same line to avoid confusion

**Comment by santiagoinfantinom** [comment]
@smartie2076, I rewrote all the paths containing variables with os.path.join and tested it at every step. The Changelog has now just one line containing all the PRs related to this and now this branch should be ready to merge

File: CHANGELOG.md [hunk 5]
```diff
@@ -17,17 +17,15 @@ Here is a template for new release sections
...
 -
-```
 ## [Unreleased]
 
 ### Added
--Constant variables for strings in constants.py (#89)
+-Constant variables for strings in constants.py (#89,#117, #118)
 
 ### Changed
 -
```
Comment (smartie2076) [inline CHANGELOG.md:26]:
```suggestion
- Constant variables for strings are gathered in `constants.py` (#89,#117, #118)

### Changed
- Replaced parameter strings with variable calls  (#89,#117, #118)
- Changed path calls to `os.path.join` to enable compatability (#118)
- Log messages now call variable of parameter name (#117, #118)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -17,17 +17,15 @@ Here is a template for new release sections
...
 ### Removed
--Constant variables for strings in constants.py (#89)
-
+-
```
Comment (smartie2076) [inline CHANGELOG.md:28]:
```suggestion
- Try/Except clauses at module import (#103) 
```
I think this is missing

**Pull request merged**